tests/components/fyta/test_config_flow.py

```diff
@@ -1,5 +1,6 @@
 """Test the fyta config flow."""
```

Comment:
Let's also add a test for the migration, where we insert a minor version 1 entry and then setup the integration, causing it to migrate to minor version 2